### PR TITLE
Fix the signing icon in the  view_list.tmpl page

### PR DESCRIPTION
**Pull request by zeripath**
There was a slight inconsistency introduced with the position of the signing avatar in #10425.

This fixes it.

### CODE REVIEW

**Comment by zeripath** [comment]
@silverwind

**Review by 6543 — APPROVED** [review]

**Comment by silverwind** [comment]
Better center for cog:

```diff
diff --git a/web_src/less/_base.less b/web_src/less/_base.less
index 1df012454..fa31b7ad4 100644
--- a/web_src/less/_base.less
+++ b/web_src/less/_base.less
@@ -1033,9 +1033,9 @@ i.icons .icon:first-child {
     margin-right: 0;
 }

 i.icon.centerlock {
-    top: 1.5em;
+    top: 10px;
 }

 .ui.label > .detail .icons {
     margin-right: .25em;
```

**Comment by silverwind** [comment]
Probably better to scope it:

```css
.sha.label i.icon.centerlock {
    top: 10px;

**Comment by silverwind** [comment]
Centering looking good with this.

**Comment by zeripath** [comment]
@silverwind are you having the cog misplaced?

**Comment by silverwind** [comment]
It's just a OCD tweak.

1.5em:

image

10px:
image

1.45em (best):
image

**Comment by zeripath** [comment]
done

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ i.icons .icon:first-child {
 }
 
 i.icon.centerlock {
-    top: 1.5em;
+    top: 10px;
```
Comment (silverwind) [inline web_src/less/_base.less]:
```suggestion
    top: 1.45em;
```

**Review by lafriks — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by guillep2k** [comment]
Ping LG-TM

**Pull request merged**